src/js/components/Timestamp.js

```diff
@@ -90,6 +93,7 @@ Timestamp.propTypes = {
     PropTypes.arrayOf(FIELD_TYPES),
     FIELD_TYPES
   ]),
+  seconds: PropTypes.bool,
```

Comment:
Agreed. I'm thinking it's in unlikely to want just seconds, or just minutes, without hours. But, this signature would support `fields=["date", "seconds"]`, as unlikely as that would be. Which I think is the right approach to the property.